### ISSUE TITLE
Identify the boot loader

### ISSUE THREAD

**Issue by pizzamaker** [KittyKatt/screenFetch#345]
Whether it's BIOS, EFI, UEFI, Coreboot or Libreboot.

**Comment by darealshinji** [comment]
Is there an easy and portable method to do that? Should it also display whether the OS is using GRUB, LILO, BTX, etc?

**Comment by pizzamaker** [comment]
Good idea! I think dmesg identifies Libreboot, but it sadly requires root.

**Comment by darealshinji** [comment]
There's a shell script available to get such information on Linux, but it also requires root or sudo to work.

**Comment by 190n** [comment]
For UEFI (and maybe EFI, I'm not sure) you can check whether `/sys/firmware/efi/efivars` is populated.

**Comment by DmitriyHetman** [comment]
How identify EFI-stub loader?

**Comment by 190n** [comment]
You can identify systemd-boot (one EFISTUB loader) when it's installed in UEFI mode by checking for the directory `/boot/EFI/systemd`.

**Comment by DmitryHetman** [comment]
also folder /boot/ visible for identifying, on almost all systems /boot is always mounted, so you can use find/grep for searching for grub,lilo etc. If nothing finded and this is amd64 system - its efi-stub kernel. Also bootloaders pointless with amd64.
